static/sass/_pattern_hardware-details-table.scss

```diff
@@ -0,0 +1,18 @@
+.p-table-hardware-details {
+  th,
+  td {
+    &:nth-child(1) {
+      width: 20%;
+      @media screen and (max-width: $breakpoint-medium) {
+        width: 28% !important;
+      }
+    }
+
+    &:nth-child(2) {
+      width: 80%;
+      @media screen and (max-width: $breakpoint-medium) {
+        width: 72% !important;
```

Comment:
Does this not work without the `!important`? CSS works in cascade, so in theory you are already overriding width when you placed the property after the initial one.

Reply:
Fixed!